src/OrdersList/Orders.cpp

```diff
@@ -0,0 +1,297 @@
+}
+
+string Bomb::getDescription() {
+    return this->description;
+}
+
+/******************************* Blockade *********************************************/
+Blockade::Blockade() {
+    this->addDescription();
+}
+
+Blockade::Blockade(Blockade* a) {
+    setOrderID(a->getOrderID());
+    this->addDescription();
+}
+
+void Blockade::execute() {
+    if (validate() == 1) cout << "Executing (Blockade) order #" << getOrderID() << " ..." << endl;
+    else cout << "Can NOT execute (Blockade) order #" << getOrderID() << " ..." << endl;
+};
+
+void Blockade::addDescription() {
+    this->description = "(Blockade) Triple the number of army units on a target territory and make it a neutral territory. "
+                        "This order can only be issued if a player has the blockade card in their hand.";
+}
+
+string Blockade::getDescription() {
+    return this->description;
+}
+
+/******************************* Airlift *********************************************/
+Airlift::Airlift() {
+    this->addDescription();
+}
+
+Airlift::Airlift(Airlift* a) {
+    setOrderID(a->getOrderID());
+    this->addDescription();
+}
+
+void Airlift::execute() {
+    if (validate() == 1) cout << "Executing (Airlift) order #" << getOrderID() << " ..." << endl;
+    else cout << "Can NOT execute (Airlift) order #" << getOrderID() << " ..." << endl;
+};
+
+void Airlift::addDescription() {
+    this->description = "(Airlift) Advance a certain number of army units from one from one territory (source territory) "
+                        "to another territory (target territory). This order can only be issued if a player has the airlift card in their hand.";
+}
+
+string Airlift::getDescription() {
+    return this->description;
+}
+
+/******************************* Negotiate *********************************************/
+Negotiate::Negotiate() {
+    this->addDescription();
+}
+
+Negotiate::Negotiate(Negotiate* a) {
+    setOrderID(a->getOrderID());
+    this->addDescription();
+}
+
+void Negotiate::execute() {
+    if (validate() == 1) cout << "Executing (Negotiate) order #" << getOrderID() << " ..." << endl;
+    else cout << "Can NOT execute (Negotiate) order #" << getOrderID() << " ..." << endl;
+};
+
+void Negotiate::addDescription() {
+    this->description = "(Negotiate) prevent attacks between the current player and another target "
+                        "player until the end of the turn. This order can only be issued if a player has the diplomacy card in their hand.";
+}
+
+string Negotiate::getDescription() {
+    return this->description;
+}
+
+/******************************* OrdersList *********************************************/
+OrdersList::OrdersList(){
+   std::list<Order*> OL;
+}
+
+void OrdersList::addOrder(Order *o){
+    OL.push_back(o);
+}
+
+//input:
+//pos is position wish to place, list starts at position 1 (1st, 2nd, etc...)
+//id of order wish to move
+bool OrdersList::move(int pos, int id){
```

Comment:
I see what u are doing, good implementation. 
i googled a bit, seems like there might be an easier way, seems to be a bit more efficient as well:
https://stackoverflow.com/questions/48330527/i-can-move-elements-within-the-stdlist-without-invalidating-iterators-nor-refe

<img width="857" alt="image" src="https://github.com/LaraMo/Risk-Swift-Sloths/assets/31352301/ac346c55-4752-4949-82ec-cd27aa88c3d8">

Reply:
Yes! I looked into documentation for splice() and swap() and it said it's for 2 lists, I thought if i applied it to the same list it would make a weird copy of the original list, and I wanted each Player to have only one list without copying it. Great insight! I can look into this at a later time!